tests/pkg_tinyvcdiff/main.c

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2022 Juergen Fitschen
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       tinyvcdiff library backend tests
+ *
+ * @author      Juergen Fitschen <me@jue.yt>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include <fcntl.h>
+#include "board.h"
+#include "embUnit.h"
+#include "vcdiff.h"
+#include "vcdiff_mtd.h"
+#include "vcdiff_vfs.h"
+#include "mtd_mapper.h"
+#include "fs/fatfs.h"
+
+static const char *bin_source = "Hello world! I hope you are doing well ...\n";
+static const char *bin_target = "Hello universe! I hope you are doing well ...\n";
+static const uint8_t diff[] = {
+    0xd6, 0xc3, 0xc4, 0x53, 0x00, 0x01, 0x2b, 0x00, 0x17, 0x2e, 0x00, 0x00,
+    0x12, 0x00, 0x0f, 0x48, 0x65, 0x6c, 0x6c, 0x6f, 0x20, 0x75, 0x6e, 0x69,
+    0x76, 0x65, 0x72, 0x73, 0x65, 0x13, 0x20, 0x0b
```

Comment:
How was this binary data created?

Reply:
Added a comment explaining how I generated this binary data.

d226cee446b1dec2b8bdd424511df4ce3950c913